storage/src/main/scala/uk/ac/wellcome/storage/tags/Tags.scala

```diff
@@ -43,7 +36,10 @@ trait Tags[Ident]
         debug(s"Tags on $id: tags have changed, so writing new tags")
         writeTags(id = id, tags = newTags) match {
           case Right(value) => Right(Identified(id, value))
-          case Left(err)    => Left(UpdateWriteError(err))
+          case Left(err) => {
+            warn(s"Error when trying to write tags to $id: $err")
```

Comment:
The other logs are structured so they look visually similar in the logs:

```suggestion
            warn(s"Tags on $id: error when trying to write: $err")
```